### PR TITLE
cleaned up homepage css

### PR DESCRIPTION
**Pull request by michaelt16**

### CODE REVIEW

**Review by dingdongg — CHANGES_REQUESTED** [review]
login form boxes look weird
image

**Review by dingdongg — COMMENTED** [review]
the little glowing effect when you hover over the inputs is gone, are we okay with that?

File: src/views/LoginView.vue [hunk 1]
```diff
@@ -73,7 +77,9 @@ img{
 .background-wrapper {
   background-color: #1F2833;
   border-radius: 5px;
-  
+  display: flex;
```
Comment (dingdongg) [inline src/views/LoginView.vue]: the flexbox stuff is already defined in line 63

Comment (michaelt16) [reply src/views/LoginView.vue]: should be good now

**Review by dingdongg — APPROVED** [review]
lgtm `728dae3cd4df3a8255f209e90f12cfb04aca7cca`

**Pull request merged**